src/utils/api.js

```diff
@@ -0,0 +1,21 @@
+const API_END_POINT = "https://kdt-frontend.programmers.co.kr";
+const X_USERNAME = 'kimhyunjoo'
+export const request = async (url='', options={}) => {
+  try {
+    const res = await fetch(`${API_END_POINT}${url}`, {
```

Comment:
이 경우 사용자가 `/foo`와 같이 슬래시까지 포함한 URL을 넣기를 기대하지만, 사람에 따라 슬래시를 안 넣는 경우가 발생할 수도 있어요 ㅎㅎ
간단한 정규식으로 URL 맨 앞에 있는 슬래시를 제거하고 직접 붙혀주는 방법으로 일관된 동작을 보장할 수 있슴다

```suggestion
    // 사용자가 입력한 값은 못 믿겠으니까 슬래시를 초기화해버리고 내가 직접 붙힌다
    const res = await fetch(`${API_END_POINT}/${url.replace(/^\/(\D.*)/, '$1')}`, {
```